src/main/java/ejercicio/banco/service/AccountServiceImpl.java

```diff
@@ -36,4 +36,20 @@ public Account findAccount(int n) {
         return account;
     }
 
+
+    //Store user
```

Comment:
What is the point of this comment? The method is super clear, `storeAccount`. 

As general, a good rule for the javadocs is: Only describe what is not clear enough with the method name + the received parameters.